### ISSUE TITLE
Crossing Management in SUMO 

### ISSUE THREAD

**Issue by ricBin93** [eclipse-sumo/sumo#14911]
In my simulation both persons and vehicles are stuck in same juction where the crossing is set.

![Screenshot 2024-05-23 125917](https://github.com/eclipse-sumo/sumo/assets/170624501/e3d7cad5-7b49-4354-b478-1da7790c3ee1)

[Ecoteckne.zip](https://github.com/eclipse-sumo/sumo/files/15415964/Ecoteckne.zip)

SUMO-version:1.19.0

operating system: Windows 11

**Comment by namdre** [comment]
The scenario involves plenty of cases with shared space but it seems some of the conflict avoidance code reacts to conflicts that don't exist.

**Comment by namdre** [comment]
@ricBin93 your network is lacking a lot of connections at junction 1741520187 and this is causing teleports.

**Comment by ricBin93** [comment]
@namdre Thanks, i fixed my net and i'm working on python code to solve the pedestrian jamming problem

[net.zip](https://github.com/eclipse-sumo/sumo/files/15436113/net.zip)

**Comment by namdre** [comment]
your scenario is still affected by #14919. Maybe wait a bit before coding a work-around.

**Comment by ricBin93** [comment]
Is it possible to assign various different emission classes in a flow of bus-type vehicles ?

**Comment by namdre** [comment]
Yes. If you define a vTypeDistribution: https://sumo.dlr.de/docs/Definition_of_Vehicles%2C_Vehicle_Types%2C_and_Routes.html#route_and_vehicle_type_distributions

**Comment by ricBin93** [comment]
Thank you, Jakob

**Comment by ricBin93** [comment]
In the HBEFA4 model related to emissions, negative values appear at certain time instances. How are these evaluated?

**Comment by namdre** [comment]
> In the HBEFA4 model related to emissions, negative values appear at certain time instances. How are these evaluated?

for anybody reading this later, the problem was resolved in see #15079
